editor.planx.uk/src/@planx/components/Section/Public.tsx

```diff
@@ -68,52 +71,85 @@ export default function Component(props: Props) {
-                  component="button"
-                  sx={{ fontFamily: "inherit", fontSize: "inherit" }}
-                >
-                  {sectionNode.data.title}
-                  <span style={visuallyHidden}>
-                    {`Change ${sectionNode.data.title}`}
-                  </span>
-                </Link>
-              ) : (
-                sectionNode.data.title
-              )}
-            </dt>
-            <dd>
-              <Tag title={sectionStatuses[sectionId]}>
-                {sectionStatuses[sectionId]}
-              </Tag>
-            </dd>
-          </React.Fragment>
-        ))}
-      </DescriptionList>
+      <SectionsOverviewList
+        sectionNodes={sectionNodes}
+        sectionStatuses={sectionStatuses}
+        showChange={true}
+        changeFirstAnswerInSection={changeFirstAnswerInSection}
+      />
     </Card>
   );
 }
 
+interface SectionsOverviewListProps {
+  sectionNodes: Record<string, SectionNode>;
+  sectionStatuses: Record<string, SectionStatus>;
+  showChange: boolean;
+  changeFirstAnswerInSection?: (sectionId: string) => void;
+}
+
+export function SectionsOverviewList(props: SectionsOverviewListProps) {
+  const {
+    sectionNodes,
+    sectionStatuses,
+    showChange,
+    changeFirstAnswerInSection,
+  } = props;
+
+  return (
+    <DescriptionList>
+      {Object.entries(sectionNodes).map(([sectionId, sectionNode]) => (
+        <React.Fragment key={sectionId}>
+          <dt>
+            {showChange &&
+            sectionStatuses[sectionId] === SectionStatus.Completed ? (
+              <Link
+                onClick={() =>
+                  changeFirstAnswerInSection &&
+                  changeFirstAnswerInSection(sectionId)
+                }
+                component="button"
+                sx={{ fontFamily: "inherit", fontSize: "inherit" }}
+              >
+                {sectionNode.data.title}
+                <span style={visuallyHidden}>
+                  {`Change ${sectionNode.data.title}`}
+                </span>
+              </Link>
+            ) : (
+              sectionNode.data.title
+            )}
+          </dt>
+          <dd>
+            <Tag title={sectionStatuses[sectionId]}>
+              {sectionStatuses[sectionId]}
+            </Tag>
+          </dd>
+        </React.Fragment>
+      ))}
+    </DescriptionList>
+  );
+}
+
 const getTagBackgroundColor = (theme: Theme, title: string): string => {
   const backgroundColors: Record<string, string> = {
-    [SectionStatus.NotStarted]: "#F3F2F1",
-    [SectionStatus.InProgress]: theme.palette.primary.main,
-    [SectionStatus.Completed]: "#00703C",
+    [SectionStatus.NotStarted]: theme.palette.grey[200],
+    [SectionStatus.InProgress]: lighten(theme.palette.primary.main, 0.9),
+    [SectionStatus.Completed]: theme.palette.success.main,
+    [SectionStatus.NeedsUpdated]: theme.palette.action.focus, // GOV UK YELLOW for now, check Figma
```

Comment:
This is much nicer / more sustainable. 

We could also add these as discrete status colours in the theme which would mean they get calculated once once the theme is generated and MUI will provide us with `contrastText` values automatically. MUI has defaults like warning, info, success etc that follow this pattern. I think this is probably only worth doing if we think these values will have meaning outside of this component though.

Docs: https://mui.com/material-ui/customization/palette/#adding-new-colors